README.md

```diff
@@ -77,6 +77,24 @@ The hot-reload is manual because you'll need to press 'r' in the terminal to hot
 
 By default, hover uses the file `lib/main_desktop.dart` as entrypoint. You may specify a different endpoint by using the `--target` flag.
 
+You can also use the configuration in the pubspec.yaml (this is the default configuration):
+```yaml
+name: ...
+description: ...
+version: ...
+
+dependencies: ...
+
+flutter: ...
+
+go-flutter:
+    target: lib/main_desktop.dart
+    branch: "@master"
```

Comment:
Should be empty string or "@latest".
[go module wiki](https://github.com/golang/go/wiki/Modules#how-to-upgrade-and-downgrade-dependencies):
> go get foo updates to the latest version of foo. go get foo is equivalent to go get foo@latest — in other words, @latest is the default if no @ version is specified.  
In this section, "latest" is the latest version with a semver tag, or the latest known commit if there are no semver tags. Prerelease tags are not selected as "latest" unless there are no other semver tags on the repository (details).